docs/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Could we just add these cmds+deps to the root level pck.json, having them separate only adds to the maintaining workload